MonstieWash/Assets/Scripts/MonsterAi/MonsterController.cs

```diff
@@ -117,7 +139,7 @@ public void TransitoryAnimationComplete()
         m_myAnimator.SetBool("mood_changed", false);
     }
 
-    public void AttackAnimationComplete()
+    public void AnimationComplete()
```

Comment:
Wondering whether it might be clearer to name this something like "InterruptAnimationComplete" now that we've got multiple actions falling under this header. At the moment it's competing with AttackAnimationComplete and simply naming it AnimationComplete might not be super obvious to Luca and Mitchell if they are ever cursed with using the Unity animator in the future. Up to you though